loco-gen/src/templates/model.t

```diff
@@ -48,7 +48,7 @@ impl MigrationTrait for Migration {
                     {% for ref in references -%}
                     .foreign_key(
                         ForeignKey::create()
-                            .name("fk-{{plural_snake}}-{{ref.0 | plural}}")
+                            .name("fk-{{plural_snake}}-{{ref.1 | plural| snake_case}}")
```

Comment:
Use `ref.0` to allow multiple references to the same table, otherwise `scaffold post created_by:references:users updated_by:references:users` would create two `fk-posts-users`.